tools/please_go/generate/generate.go

```diff
@@ -30,9 +30,10 @@ type Generate struct {
 	knownImportTargets map[string]string // cache these so we don't end up looping over all the modules for every import
 	thirdPartyFolder   string
 	install            []string
+	extraLabels        []string
```

Comment:
why is this called `extraLabels` and not just `labels`?

Reply:
I wasn't sure whether there were a set of base labels that would be applied to the target besides these. I will rename if not.